examples/README.md

```diff
@@ -0,0 +1,31 @@
+# ink! code examples
+
+This folder contains a set of example contracts for ink!.
+
+Have a look at the different examples to better understand how to use ink! to build your own Substrate smart contracts.
+
+## Preparation
+
+For building the example smart contracts found in this folder you will need to have `cargo-contract` installed.
+
+```
+cargo install --git https://github.com/paritytech/ink cargo-contract
+```
+
+Use the `--force` option to update to the most recent `cargo-contract` version.
```

Comment:
```suggestion
We use the `--force` to update to the most recent `cargo-contract` version.
```

Reply:
Is that some kind of Star Wars reference?